x/hard/keeper/liquidation.go

```diff
@@ -22,7 +22,7 @@ func (k Keeper) AttemptIndexLiquidations(ctx sdk.Context) error {
 	borrowers := k.GetLtvIndexSlice(ctx, params.CheckLtvIndexCount)
 
 	for _, borrower := range borrowers {
-		_, err := k.AttemptKeeperLiquidation(ctx, sdk.AccAddress(types.LiquidatorAccount), borrower)
+		err := k.AttemptKeeperLiquidation(ctx, sdk.AccAddress(types.LiquidatorAccount), borrower)
 		if err != nil {
 			if !errors.Is(err, types.ErrBorrowNotLiquidatable) {
```

Comment:
```
!errors.Is(err, types.ErrBorrowNotLiquidatable) && !errors.Is(err, types.ErrBorrowNotFound) {
```

catch case where borrower doesn't exist (ie, that user has only supplied coins)

Reply:
Fixed in https://github.com/Kava-Labs/kava/pull/758/commits/52d86dfca41c429246af6dc2356a96101f936e9f